Makefile

```diff
@@ -390,10 +390,13 @@ template-in-dispvm-%:
 
 # Sign only unsigned files (naturally we don't expect files with WRONG sigs to be here)
 COMPONENTS_TO_SIGN := $(if $(NO_SIGN),,$(COMPONENTS))
-.PHONY: sign-all sign-dom0 sign-vm
+.PHONY: sign-all sign-dom0 sign-vm sign-iso
 sign-all:: $(COMPONENTS_TO_SIGN:%=sign.%);
 sign-dom0:: $(COMPONENTS_TO_SIGN:%=sign.dom0.%);
 sign-vm:: $(COMPONENTS_TO_SIGN:%=sign.vm.%);
+sign-iso: ISO_VERSION=$(shell cat $(SRC_DIR)/$(INSTALLER_COMPONENT)/build/ISO/qubes-x86_64/iso/build_latest)
+sign-iso:
+	$(BUILDER_DIR)/scripts/release-iso iso/Qubes-DVD-x86_64-$(ISO_VERSION).iso
```

Comment:
This is legacy file name, the new one is `Qubes-${version}-x86_64.iso`. `release-iso` script will rename it. Maybe it would be logical to rename it earlier - like at the end of the iso build?

Reply:
Yes.